runtime.c

```diff
@@ -385,6 +392,38 @@ main (int argc, char *argv[])
         exit(0);
     }
 
+    if (arg && strcmp(arg,"appimage-portable-home")==0) {
+        char portable_home[2048];
```

Comment:
We use `LINE_SIZE` in other places - how did you determine 2048?

Reply:
I used the same value used for `portable_home_dir` and `portable_config_dir`